lalrpop/src/grammar/repr.rs

```diff
@@ -144,7 +144,7 @@ pub enum ActionFnDefnKind {
 /// An action fn written by a user.
 #[derive(Clone, PartialEq, Eq)]
 pub struct UserActionFnDefn {
-    pub arg_patterns: Vec<Atom>,
+    pub arg_patterns: Vec<(bool, Atom)>,
```

Comment:
(IMO) Especially here, but also in the other field it would be more readable with

```rust
enum Mutability { Mutable, Immutable }
```

Reply:
Hmm, I think it's a @ahmedcharles's commit artifact I accidentally didn't notice. I believe we could just use a new `Name` here